mpa/det/trainer.py

```diff
@@ -153,7 +155,12 @@ def train_worker(gpu, target_classes, datasets, cfg, distributed=False,
         # model
         model = build_detector(cfg.model)
         model.CLASSES = target_classes
-
+        # Do clustering for SSD model
+        if hasattr(cfg.model, 'bbox_head') and hasattr(cfg.model.bbox_head, 'anchor_generator'):
```

Comment:
How about `cfg.model.get('bbox_head', False)` following the convention of other tasks?

Reply:
I just followed mmdetection's rule.

Reply:
I think the original code looks better.